javascript/javascript3/week1/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <input type="text" textholder="write city name">
+    <button class="submit">Weather</button>
+    <button id="currentPosition">Get current position</button>
```

Comment:
Just a side note : The camelCase is preferred way of naming your variables and functions in JS, however the convention for html would be to name your html attributes either all in lower case or connected with a "-". 👩🏻‍🎨

```
<button id="current-position"></>
```